tutorials/clustering.md

```diff
@@ -22,15 +24,93 @@ need a common superuser role on each postgres instance. This tutorial
 will assume the `postgres` role for this, but you may substitute any
 role common to all machines with superuser permission.
 
-Beyond this, please ensure that you've completed the following steps.
-Some or most of these may not be necessary depending on how you
-initially set up your instances:
-- Make sure you've enabled `max_prepared_transactions` on all the instances, this parameter is located in `postgresql.conf`. We recommend a setting of at least 150.
-- Make sure the data nodes are properly listening for network traffic. `postgresql.conf` should have a `listen_addresses` set to include the access node. If this parameter is missing or just `localhost`, change it to `access.example.com`, or the corresponding IP. You can also set it to `"*"` to not restrict where connections can originate from.
-- The superuser role, `postgres` in this example, must have an entry in the `pg_hba.conf` file for the data nodes. This should look something like: ```host    all    postgres    access.example.com    trust``` You can use any CIDR block covering the access node's IP in place of `access.example.com`. For using authentication mechanism's other than `trust`, please see the [authentication guide][data-node-authentication].
+### Update your Postgres configuration
+
+First of all, if you haven't yet run the `timescaledb-tune` utility, it's
+highly recommended that you do so before starting. Do this for all of your
+nodes.
+```
+sudo timescaledb-tune
+```
+
+In addition, make sure you've enabled prepared transactions on your data
+nodes. This is essential for allowing Timescale distributed transactions to
+work. prepared transactions are disabled by default, so make sure you have
+this line in `postgresql.conf` on every data node.
+```
+max_prepared_transactions = 150
+```
+
+### Set password encryption
+
+You are now ready to set up the network communication between the nodes and
+add authentication. For more details and option for this step, please visit
+the [data node authentication][data-node-authentication] page.
 
-After making any of these changes, please restart your Postgres instance to ensure the
-new settings take effect.
+In this example you'll be adding `scram-sha-256` password authentication to your
+nodes. The first step of this is to enable password encryption on all the nodes
+by adding the following to `postgresql.conf`:
+
+```
+password_encryption = 'scram-sha-256'
+```
+
+>:WARNING: This setting is not automatically applied to any existing passwords.
+Any existing roles will have to update to new passwords to take advantage of
+the encryption. You can do this in Postgres by using the `ALTER ROLE` command
+to specify the new password (this can be the same as the old password).
```

Comment:
This is described in better details in the section to set up the data nodes. May be give a link  instead?

Reply:
The example in that section is actually intended to solve the issue of the postgres role not having a password set up.  I'm not sure it would make sense to the user to point them at that section, though I did add another sentence there explaining how to fix the problem if they run the command too soon.  For this section, I added a link to the postgres ALTER ROLE documentation.